src/gcp_scanner/crawl.py

```diff
@@ -822,8 +818,8 @@ def get_app_services(project_name: str,
     app_services["services"] = list()
     while request is not None:
       response = request.execute()
-      for service_entry in response.get("services", []):
-        app_services["services"].append(service_entry)
+      app_services['services'] = [service_entry 
+                                  for service_entry in response.get("services", [])]
```

Comment:
Redundant.

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.

Reply:
This is still not fixed.